data/issues.py

```diff
@@ -0,0 +1,88 @@
+import logging
+
+import requests
+
+from data.models import (
+    Issue,
+    Label,
+    )
+from data.newcomers import get_newcomers
+from data.models import Contributor
+from data.web import web_url
+
+
+def fetch_issues(hoster):
+    """
+    Get issues opened by newcomers
+
+    :param hoster: a string representing hoster, e.g. 'github'
+    :return: a json of issues data
+    """
+    logger = logging.getLogger(__name__)
+    if hoster == 'github':
+        IMPORT_URL = web_url + 'issues/github/all'
+    elif hoster == 'gitlab':
+        IMPORT_URL = web_url + 'issues/github/all'
+    else:
+        IMPORT_URL = 'https://pastebin.com/raw/VgYXYDEN'
+    headers = {'Content-Type': 'application/json'}
+    try:
+        response = requests.get(
+            url=IMPORT_URL,
+            headers=headers,
+        )
+        response.raise_for_status()
+    except Exception as e:
+        logger.error(e)
+        return
+    issues = response.json()
+
+    # Removing issues which are not opened by newcomers
+    for issue in issues[:]:
+        if issue['author'] not in get_newcomers():
+            issues.remove(issue)
+    return issues
+
+
+def import_issue(hoster, issue):
+    """
+    Import issue data to database
+
+    :param hoster: a string representing hoster
+    :param issue: a dict containing issue's data
+    """
+    logger = logging.getLogger(__name__)
+    number = issue.get('number')
+    assignees = issue.pop('assignees')
+    labels = issue.pop('labels')
+    author = issue.pop('author')
+    try:
+        c = Contributor.objects.get(login=author)
+        issue['author'] = c
+        issue['hoster'] = hoster
+        i, created = Issue.objects.get_or_create(
+            **issue
+            )
+        if created:
+            i.save()
```

Comment:
Same as previous comment, this is not needed. See https://docs.djangoproject.com/en/2.0/ref/models/querysets/#get-or-create
`If an object is not found, get_or_create() will instantiate and save a new object, returning a tuple of the new object and True`

Reply:
There are many other places in this pull request which explicitly call `save` method after `get_or_create`, which is redundant.